contracts/drafts/ERC20Permit.sol

```diff
@@ -62,7 +62,7 @@ abstract contract ERC20Permit is ERC20, IERC20Permit, EIP712 {
     /**
      * @dev See {IERC20Permit-nonces}.
      */
-    function nonces(address owner) public view override returns (uint256) {
+    function nonces(address owner) public view virtual override returns (uint256) {
```

Comment:
What do you think about this one given that we also do `_nonces[owner].increment()`?